docs/cheatsheet/cheatsheet.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<!--
+Only renders in Chrome.
+
+cheatsheet.pdf: Save as PDF from Chrome
+cheatsheet.png: Save as image from Chrome using https://chrome.google.com/webstore/detail/full-page-screen-capture/fdpohaocaechififmbbbbbknoalclacl
+-->
+<html>
+<head>
+<meta http-equiv="content-type" content="text/html; charset=UTF-8">
+<link rel="stylesheet" type="text/css" href="cheatsheet.css">
+</head>
+<body>
+
+<div class="header">
+<div class="usage">
+<span class="one">EarlGrey().selectElementWithMatcher</span>(<span class="two">ViewMatcher</span>)
+                      .performAction(<span class="three">ViewAction</span>)
+                      .assertWithMatcher(<span class="two">ViewMatcher</span>)
+</pre>
+</div>
+
+<div class="cheatsheet">
+EarlGrey
```

Comment:
Unless we add a build system that pulls the version from the source code, hardcoding is the path of least resistance.

Fixed.